### PR TITLE
chore(ci): Adds autoconfig ci test

### PR DESCRIPTION
**Pull request by dmihalcik-virtru**

### CODE REVIEW

File: examples/cmd/attributes.go [hunk 1]
```diff
@@ -46,6 +46,18 @@ func init() {
...
 	attributes.AddCommand(add)
 
+	assign := &cobra.Command{
+		Use:  "assign",
+		Args: cobra.NoArgs,
+		RunE: func(cmd *cobra.Command, args []string) error {
+			return assignAttribute(cmd, true)
+		},
+	}
+	assign.Flags().StringVarP(&attr, "attr", "a", "", "attribute prefix, e.g. http://name.space/attr/name")
```
Comment (pflynn-virtru) [inline examples/cmd/attributes.go:56]:
```suggestion
	assign.Flags().StringVarP(&attr, "attr", "a", "", "attribute prefix, e.g. https://name.space/attr/name")
```

Comment (pflynn-virtru) [reply examples/cmd/attributes.go:56]: I think we enforce `https://`

File: examples/cmd/attributes.go [hunk 2]
```diff
@@ -46,6 +46,18 @@ func init() {
...
+	assign.Flags().StringVarP(&kas, "kas", "k", "", "which kas to assign")
+	assign.Flags().StringVarP(&values, "values", "v", "", "any attibute values to include; if empty, applies to all")
```
Comment (pflynn-virtru) [inline examples/cmd/attributes.go:58]:
```suggestion
	assign.Flags().StringVarP(&values, "values", "v", "", "any attribute values to include; if empty, applies to all")
```

File: examples/cmd/attributes.go [hunk 3]
```diff
@@ -72,6 +84,18 @@ func init() {
...
 	attributes.AddCommand(remove)
 
+	unassign := &cobra.Command{
+		Use:  "unassign",
+		Args: cobra.NoArgs,
+		RunE: func(cmd *cobra.Command, args []string) error {
+			return assignAttribute(cmd, false)
+		},
+	}
+	unassign.Flags().StringVarP(&attr, "attr", "a", "", "attribute prefix, e.g. http://name.space/attr/name")
```
Comment (pflynn-virtru) [inline examples/cmd/attributes.go:94]:
```suggestion
	unassign.Flags().StringVarP(&attr, "attr", "a", "", "attribute prefix, e.g. https://name.space/attr/name")
```

File: examples/cmd/attributes.go [hunk 4]
```diff
@@ -72,6 +84,18 @@ func init() {
...
+	unassign.Flags().StringVarP(&kas, "kas", "k", "", "which kas to assign")
+	unassign.Flags().StringVarP(&values, "values", "v", "", "any attibute values to include; if empty, applies to all")
```
Comment (pflynn-virtru) [inline examples/cmd/attributes.go:96]:
```suggestion
	unassign.Flags().StringVarP(&values, "values", "v", "", "any attribute values to include; if empty, applies to all")
```

File: examples/cmd/attributes.go [hunk 5]
```diff
@@ -232,7 +256,7 @@ func addAttribute(cmd *cobra.Command) error {
 func removeAttribute(cmd *cobra.Command) error {
 	s, err := newSDK()
 	if err != nil {
-		slog.Error("could not connect", slog.String("error", err.Error()))
+		slog.Error("could not connect", "err", err)
```
Comment (pflynn-virtru) [inline examples/cmd/attributes.go:259]: unknown which approach to do here.  I think Sean likes the original

File: examples/cmd/attributes.go [hunk 6]
```diff
@@ -304,6 +328,135 @@ func removeAttribute(cmd *cobra.Command) error {
...
+	if err != nil {
+		return err
+	}
+
+	kasById := make(map[string]string)
+
+	var kasids []string
+	switch {
+	case kas != "":
+		kasid, err := upsertKAS(cmd.Context(), s, kas, nil)
```
Comment (pflynn-virtru) [inline examples/cmd/attributes.go:363]: `upsertKAS` not to be confused with KAS `upsert` remote policy

**Review by pflynn-virtru — APPROVED** [review]

File: examples/cmd/attributes.go (hunk 5 above)
Comment (dmihalcik-virtru) [reply examples/cmd/attributes.go:259]: yeah I'll do a follow up PR at some point that makes these consistent

**Review by pflynn-virtru — APPROVED** [review]

**Pull request merged**